tests/neg/i2514.scala

```diff
@@ -1,10 +1,28 @@
 object Foo {
   def foo(): Int = {
-    val f: implicit Int => Int = (implicit x: Int) => 2 * x // error // error
+    val f: implicit Int => Int = (implicit x: Int) => 2 * x // error
```

Comment:
This change is cause by the use of `em` instead of `hl`, which, IIUC, will shadow non nonsensically errors after the first one.  On master this line has an additional error: `// error: only classes can have declared but undefined members`, I think keeping only the first one makes sense here.